apiary.apib

```diff
@@ -4142,10 +4142,10 @@ Get a list of tracked time.
     + Attributes (object)
         + filter (object, optional)
             + user_id: `87982c96-f2fe-4b05-838c-ff42c0525758` (string, optional)
-            + started_after: `2017-04-26T10:01:49+00:00` (string, optional)
-            + started_before: `2017-04-26T10:01:49+00:00` (string, optional)
-            + ended_after: `2017-04-26T10:01:49+00:00` (string, optional)
-            + ended_before: `2017-05-26T10:01:49+00:00` (string, optional)
+            + started_after: `2017-04-26T10:01:49+00:00` (string, optional) - Start of the period for which to return time tracking. This includes time tracking that started on the date passed.
```

Comment:
I would not use the word "passed", but rather "provided". Since "the date passed" has a potential different meaning.

Reply:
Yea makes sense. updated